Signal/src/UserInterface/Strings.swift

```diff
@@ -15,16 +15,13 @@ import Foundation
 
```

Comment:
Oh, so you see multiple classes being defined in this single source file. Got it.